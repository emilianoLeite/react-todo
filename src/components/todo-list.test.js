import React from "react";
import { shallow } from "enzyme";
import TodoList from "./todo-list";

it("renders without crashing", () => {
  shallow(<TodoList />);
});
