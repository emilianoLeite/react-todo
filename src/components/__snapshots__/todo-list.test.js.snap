// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when some todos are supplied matches snapshot 1`] = `
<ul>
  <li
    className=""
  >
    todo1
  </li>
  <li
    className="completed"
  >
    todo2
  </li>
</ul>
`;
